src/main/resources/com/microsoft/azure/util/AzureCredentials/credentials.jelly

```diff
@@ -4,17 +4,17 @@
     <j:set var="uniqueId" value="${h.generateId()}"/>
     <f:entry title="${%SubscriptionID}" field="subscriptionId"
              help="/plugin/azure-credentials/help-subscriptionId.html">
-        <f:password/>
+        <f:textbox/>
     </f:entry>
     <f:entry title="${%ClientID}" field="clientId" help="/plugin/azure-credentials/help-clientId.html">
-        <f:password/>
+        <f:textbox/>
     </f:entry>
     <f:entry title="${%ClientSecret}" field="clientSecret" help="/plugin/azure-credentials/help-clientSecret.html">
         <f:password/>
     </f:entry>
     <f:entry title="${%OAuthToken}" field="oauth2TokenEndpoint"
              help="/plugin/azure-credentials/help-oauth2TokenEndpoint.html">
-        <f:password/>
+        <f:textbox default="${descriptor.getDefaultOAuthPrefix()}"/>
```

Comment:
I think they're different for that some fields are plain text and others are encoded cipher in /var/lib/jenkins/credentials.xml